tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from utils.singleton import destroyAllSingletonInstances
+from utils.singleton import noSingletonsAround
+
+@pytest.fixture(scope="function", autouse=True)
+def my_fixture(): # pylint: disable=invalid-name
```

Comment:
Something I'll look into at merge time... for autouse, does that apply to the built-in tests in robotpy? I assume yes, but then I also worry/question if that stomps on some functionality they assume.

Find as is for sure, but something to keep in mind. Narrowing scope to the non-pytest-built-in tests may be safer.